plugins/txt/plugin/commands/txt_reply_cmd.rb

```diff
@@ -0,0 +1,38 @@
+module AresMUSH
+  module Txt 
+      class TxtReplyCmd
+        include CommandHandler
+
+        attr_accessor :names, :names_raw, :message, :scene_id
+
+    def parse_args
+      if (!cmd.args)
```

Comment:
FYI you can just do `self.message = cmd.args`.  If cmd.args is nil, it'll set message to nil too.

Reply:
Fixed.